pkgs/misc/lilypond/update.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -I nixpkgs=./. -i bash -p curl gnused nix
+
+set -euo pipefail
+
+if [ $# -gt 0 ] && [ "$1" = "unstable" ]; then
+    ATTR="lilypond-unstable"
+    FILE="$(dirname "${BASH_SOURCE[@]}")/unstable.nix"
+    QUERY="VERSION_DEVEL="
+else
+    ATTR="lilypond"
+    FILE="$(dirname "${BASH_SOURCE[@]}")/default.nix"
+    QUERY="VERSION_STABLE="
+fi
+
+# update version
+PREV=$(nix eval --raw -f default.nix $ATTR.version)
+NEXT=$(curl -s https://raw.githubusercontent.com/lilypond/lilypond/master/VERSION | grep "$QUERY" | cut -d= -f2)
```

Comment:
should we prefer https://git.savannah.gnu.org/gitweb/?p=lilypond.git or https://gitlab.com/lilypond/lilypond instead?

ref: https://lilypond.org/development.html